packages/client/components/PokerEstimateHeaderCardHtml.tsx

```diff
@@ -74,16 +71,19 @@ const StyledLabel = styled('span')({
 })
 
 interface Props {
-  issue: PokerEstimateHeaderCardJira_issue
+  summary: string
+  descriptionHTML: string
+  url: string
+  linkTitle: string
+  linkText: string
 }
-const PokerEstimateHeaderCardJira = (props: Props) => {
-  const {issue} = props
+const PokerEstimateHeaderCardHtml = (props: Props) => {
```

Comment:
+1 the github card is just HTML, too, so the name could be a little more discriminatory.

perhaps if we rename `JiraIssue` to `JiraCloudIssue` and then make `JiraCloudIssue` and `JiraServerIssue` implement a `JiraIssue` interface?

Reply:
@mattkrick Renamed `PokerEstimateHeaderCardHtml` to `PokerEstimateHeaderCardContent`

What if we make a single interface with unified fields for all integrations? Something like `IntegrationIssue`.
For example, a common `summary` field resolves to `title` for GitHub and to `summary` for Jira and Jira Server
What do you think?

Reply:
that's exactly how it used to work! https://github.com/ParabolInc/parabol/blob/f77f81a4935f3e7bd9e29f77ebc798e757a71710/packages/server/graphql/types/JiraIssue.ts#L45-L51

i had to get rid of it because there was no way to tell the client that the 2 fields were the same. for example, say a component requests `title`, then you navigate to another component that requires `summary`. normally it could use the cached value, but since they have different names, it'd have to make a separate network request because it saw that `summary` was a missing field.

Similarly, when creating an issue in an optimistic updater, you had to remember to populate both fields otherwise you could get an error. 

If 5 integrations all call the same field something different, that'd be 5 extra fields we'd have to remember to request, which just seemed like it was ripe for bugs. We could instead replace e.g. `summary` with `title`, but then that would get confusing when making the request to the integration.

I think the _right_ way to do it is by adding that field on the _client_, not the server, via a clientSchema extension. Then, you can have a client resolver that resolves 1 value to the other. If this sounds interesting to you, you can experiment with it, the details on how to do it are here: https://relay.dev/docs/guided-tour/reusing-cached-data/filling-in-missing-data/

Reply:
@mattkrick I feel like it is more robust and easier to implement if the backend exposes ready-to-use fields. 
As we would need to pick some approach and made some refactoring, I created a separate discussion issue here #6374 and wrote my concerns and proposal with more details. What do you think?